frontend/northeast-rivers-explorer/style.css

```
@@ -0,0 +1,339 @@
+:root {
+    --bg: #0c1614;
+    --bg-alt: #12201c;
+    --surface: #1a2c28;
+    --line: #2e4a42;
+    --text: #eef6f2;
+    --muted: #b8d0c6;
+    --dim: #7e9a90;
+    --accent: #5ec4a8;
+    --accent-2: #d4b06a;
+    --heading: "Playfair Display", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)